RecoLocalMuon/GEMRecHit/python/me0RecHits_cfi.py

```diff
@@ -3,5 +3,5 @@
 me0RecHits = cms.EDProducer("ME0RecHitProducer",
     recAlgoConfig = cms.PSet(),
     recAlgo = cms.string('ME0RecHitStandardAlgo'),
-    me0DigiLabel = cms.InputTag("simMuonME0ReDigis"),
+    me0DigiLabel = cms.InputTag("simMuonME0PseudoReDigis"),
```

Comment:
For my education, could you briefly summarize "pseudo" and "re"? (pseudo means smeared, and "re" means?) -perhaps the word "sim" needs to be dropped at some point along this chain